_common

```diff
@@ -39,7 +39,7 @@ runcli() {
     output=$("${args[@]}" 2>&1)
     rc=$?
     set -e
-    [[ "$rc" == 0 ]] && echo "$output" && return
+    [[ $rc -eq 0 ]] && echo "$output" && return
```

Comment:
I honestly knew about the subtleties of `==` vs. `eq` in bash but still prefered `==` as a more readable variant. You wrote what we *should* use but if there isn't a more compelling reason I'd prefer to just keep the "=="

Reply:
I suppose if shellcheck isn't griping then it's very likely ok.
I hadn't thought about using `==` for numbers before but I can't yet think of a reason it wouldn't work.
In that case I'd stick with it. I'll let you know if I find any edge cases.
I might start using this as well if it's safe. :)